terraso_backend/apps/core/management/commands/harddelete.py

```diff
@@ -0,0 +1,55 @@
+from datetime import datetime, timedelta, timezone
+
+import structlog
+from django.apps import apps
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.db import models
+from safedelete.models import HARD_DELETE
+
+logger = structlog.get_logger(__name__)
+
+
+class Command(BaseCommand):
+
+    DEFAULT_DELETION_GAP = timedelta(days=settings.HARDDELETE_DELETION_GAP)
+
+    help = "Hard delete rows in database that were soft-deleted before a specified time"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--exec_time",
```

Comment:
issue: still find this name confusing.

Reply:
I added a help statement below to explain what this does. I'm not sure how to give this a more descriptive name, do you have any suggestions?

Reply:
I'm confused as to why there are two arguments. You are providing both an offset and a gap? So you can say "30 days from yesterday"?

Reply:
i also think exec_time is a bit confusing when not viewed from the perspective of testing the code, it makes me think it's an argument for scheduling when the deletion will run. if it is not useful outside of testing, could we either manually compute the desired deletion gap in the test code or at least omit `--exec_time` from the command line arguments so it doesn't confuse someone trying to use the command?

Reply:
>  if it is not useful outside of testing, could we either manually compute the desired deletion gap in the test code or at least omit --exec_time from the command line arguments so it doesn't confuse someone trying to use the command?

@paulschreiber do you think I should just remove the argument? I will have to rewrite the test, which will take additional time.

Reply:
@david-code Yes. I agree with my previous statement that having two arguments is confusing.